norse/torch/functional/spikes_to_times_decoder.py

```diff
@@ -0,0 +1,98 @@
+import torch
+
+
+class SpikesToTimesDecoder(torch.nn.Module):
+    """
+    Module wrapper for spike to spike-time (or spike-index) decoder.
+
+    Decodes input spikes with shape [time_steps x batch_size x output_size] into spike indices
+    tensor with shape [spike_count x batch_size x output_size] in ascending order, e.g. for
+    spike_count = 1 returns only times of first spikes along each batch- and output-dimension.
+    If stated, converts indices along time axis into times with given dt value.
+    """
+
+    def __init__(
+        self,
+        spike_count: int = 1,
+        convert_indices_to_times: bool = True,
+        dt: float = 1e-3,
+    ):
+        """
+        Initialize decoder.
```

Comment:
Do these docstrings generate ok? I haven't tried to split up module and __init__ function documentation like this before.

Reply:
I often used them like this, didn't lead to any problems yet

Reply:
Gotcha. I tried to generate the docs, and they're failing for some reason. None of the classes appear to have their own pages:
![image](https://user-images.githubusercontent.com/1064317/167860556-3fb1f5dd-f575-43f6-8d1d-a2a4720deb3e.png)

I'm happy to defer this to another issue - we need to clean up our documentation anyway :-)